webapp/src/cljs/lipas/ui/map/utils.cljs

```diff
@@ -118,6 +119,16 @@
       )
   )
 
+(defn simplify-scale [n]
+  (* 0.00001 n))
```

Comment:
Maybe a comment could clarify this magic number?

Reply:
Good suggestion! Added docstring